kafka/producer/base.py

```diff
@@ -27,6 +27,9 @@
 
 STOP_ASYNC_PRODUCER = -1
 
+# unlimited
+ASYNC_QUEUE_MAXSIZE = 0
```

Comment:
I got your point. It's fine to me, I was just trying to reduce the expected behavior of the current ``master`` branch. I'll set the suggested limit.